src/common.rs

```diff
@@ -764,6 +765,32 @@ pub fn print_unsigned_transaction(transaction: &crate::commands::PrepopulatedTra
                 };
                 print_unsigned_transaction(&prepopulated_transaction);
             }
+            near_primitives::transaction::Action::DeployGlobalContract(deploy) => {
+                let code_hash = CryptoHash::hash_bytes(&deploy.code);
+                let identifier = match deploy.deploy_mode {
+                    GlobalContractDeployMode::CodeHash => {
+                        format!("deploy code <{:?}> as a global hash", code_hash)
+                    }
+                    GlobalContractDeployMode::AccountId => {
+                        format!(
+                            "deploy code <{:?}> to a global account <{}>",
+                            code_hash, transaction.receiver_id
```

Comment:
for the purpose of symmetry + clarity, lines 656-662 for `near_primitives::transaction::Action::DeployContract(code)`  should mention which account id
`code_hash` is deployed too (` to account <{}>`)

Reply:
Not sure if there would be a link to account in that scenario.

Reply:
well, it's computed from somewhere, probably from `transaction.receiver_id` too. Otherwise, where does it come from, if the target account is not included into the action either

Reply:
I mean after deploy not sure if it will be a part of user state. Because the doc day that it would be another trie on shard level with a copy in each shard.

Reply:
nope, not getting what you mean at all. 
I meant adding it here https://github.com/near/near-cli-rs/blob/main/src/common.rs#L655-L662 , 
for the regular deploy, not a global deploy to hash